model-optimizer/mo/middle/passes/mean_scale_values_test.py

```diff
@@ -38,137 +38,239 @@
-                             ('placeholder_1_data', 'scaleshift_1'),
-                             ('scaleshift_1', 'scaleshift_1_data'),
-                             ('scaleshift_1_w', 'scaleshift_1'),
-                             ('scaleshift_1_b', 'scaleshift_1'),
-                             ('scaleshift_1_data', 'op_output')
+                             ('placeholder_1_data', 'mul'),
+                             ('mul', 'mul_output'),
+                             ('const_mul', 'const_mul_output'),
+                             ('const_mul_output', 'mul'),
+                             ('mul_output', 'op_output'),
                              ],
                             {'placeholder_1_data': {'shape': np.array([1, 227, 227, 3])},
-                             'scaleshift_1_w': {'shape': np.array([3]), 'value': np.ones(3)},
-                             'scaleshift_1_b': {'shape': np.array([3]), 'value': np.zeros(3)},
+                             'const_add_output': {'shape': np.array([3]), 'value': np.array([-1, -2, -3])},
                              })
         graph.graph['cmd_params'] = Namespace(reverse_input_channels=False)
-        del graph['placeholder_1']['placeholder_1_data'][0]['in']
-        del graph['scaleshift_1']['scaleshift_1_data'][0]['in']
 
         graph_ref = build_graph(nodes_attributes,
-                                [('placeholder_1', 'scaleshift_1_data'),
-                                 ('scaleshift_1_data', 'op_output')
+                                [('placeholder_1', 'placeholder_1_data'),
+                                 ('placeholder_1_data', 'op_output'),
                                  ])
 
-        move_scaleshift_to_preprocess(graph)
+        move_mean_values_to_preprocess(graph)
         self.assertTrue(graph.graph.get('mean_values', None) is None)
+        # self.assertTrue(np.array_equal(graph.graph['mean_values']['placeholder_1'], np.array([1, 2, 3])))
 
-        (flag, resp) = compare_graphs(graph, graph_ref, 'scaleshift_1_data')
+        (flag, resp) = compare_graphs(graph, graph_ref, 'op_output')
         self.assertTrue(flag, resp)
+
+
+    def test_move_mean_values_to_preprocess_7(self):
+        graph = build_graph(nodes_attributes,
+                            [('placeholder_1', 'placeholder_1_data'),
+                             ('placeholder_1_data', 'add'),
+                             ('add', 'add_output'),
+                             ('const_add', 'const_add_output'),
+                             ('const_add_output', 'add'),
+                             ('add_output', 'op_output'),
+                             ],
+                            {'placeholder_1_data': {'shape': np.array([1, 227, 227, 3])},
+                             'const_add_output': {'shape': np.array([3]), 'value': np.array([-1, -2, -3])},
+                             })
+        graph.graph['cmd_params'] = Namespace(reverse_input_channels=False)
```

Comment:
All tests cover case when `reverse_input_channels` are `False`. Please, add a case when it is equal to `True`